<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QGraphicsView" name="graphicsView_5"/>
    </item>
    <item row="4" column="0">
     <widget class="QGraphicsView" name="graphicsView_4"/>
    </item>
    <item row="4" column="2">
     <widget class="QGraphicsView" name="graphicsView_6"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGraphicsView" name="graphicsView_2"/>
    </item>
    <item row="0" column="2">
     <widget class="QGraphicsView" name="graphicsView_3"/>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGraphicsView" name="graphicsView_1"/>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
